src/interfaces/filing-interfaces/index.ts

```diff
@@ -5,3 +5,6 @@ export * from './filingData-interface'
 export * from './filingHeader-interface'
 export * from './incorporationApplication-interface'
 export * from './incorporationFiling-interface'
+
+// Proxy export for Shared Components which rely on IncorporationAddressIf
+export { AddressesIF as IncorporationAddressIf } from '@/interfaces/stepper-interfaces/YourCompany/address-interface'
```

Comment:
Sad but OK. (We _should_ go and update the shared components, but it's a breaking change.)

This is more of an alias than a proxy :)

Reply:
hahah i'm not sure! Alias sounds deceitful

Reply:
This is how I understand it:
Proxy: something on behalf of something else.
Alias: another name for something (*).

(*) Only deceitful in legal circles 😆

Reply:
hahaha enjoying this comment!

ok, i had to post these here. Out of context and english has a thousand meanings for every word.
Updated to alias tho :) 

![Screen Shot 2022-03-08 at 1 59 05 PM](https://user-images.githubusercontent.com/53542131/157332810-e0d14ff2-64ab-42ec-aa62-20ccd45f26e6.png)
![Screen Shot 2022-03-08 at 1 58 48 PM](https://user-images.githubusercontent.com/53542131/157332812-e1582ad7-eb83-4272-b072-a93642178f62.png)

Reply:
Updated